dev/tools/update_icons.dart

```diff
@@ -332,14 +330,14 @@ class _Icon {
     id = tokenPair.key;
     hexCodepoint = tokenPair.value;
 
-    if (id.endsWith('_outlined') && id!='insert_chart_outlined') {
-      shortId = _replaceLast(id, '_outlined');
+    if (id!.endsWith('_outlined') && id!='insert_chart_outlined') {
```

Comment:
Might as well clean this up a little:
```suggestion
    if (id!.endsWith('_outlined') && id != 'insert_chart_outlined') {
```